src/windows.hpp

```diff
@@ -51,6 +51,7 @@
 #include <winsock2.h>
 #include <windows.h>
 #include <mswsock.h>
+#include <iphlpapi.h>
```

Comment:
hmm, just checked and the function i added it for isn't even in this build... must have gotten confused when patching the development branch.